map_view/display-device-location-with-autopan-modes/README.metadata.json

```
@@ -0,0 +1,37 @@
+{
+    "category": "Map",
+    "description": "Display your current position on the map, as well as switch between different types of auto pan modes.",
```

this doesn't match the description in the above readme?

```suggestion
    "description": "Display your position on the map and switch between different types of autopan modes.",
```